google/cloud/aiplatform/helpers/container_uri_builders.py

```diff
@@ -107,3 +108,17 @@ def get_prebuilt_prediction_container_uri(
         )
 
     return final_uri
+
+
+def is_prebuilt_prediction_container_uri(image_uri: str) -> bool:
```

Comment:
Is this just used to determine the python and pip command?

Reply:
For now, yes.